saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/SpatialOntologyDataModel.scala

```diff
@@ -57,6 +57,13 @@ object SpatialOntologyDataModel extends DataModel {
     x: Token => ""
   }
 
+  val spatialRole = property(tokens)
+  {
+    x: Token => x.getPropertyValues("Trajector").get(0)
+  }
+  //Question: how I can get the information that I have in the XML file tags here as the properties of the base classes.
+  //This should be added to the documentation. 
+
```

Comment:
instead of `x.getPropertyValues("Trajector").get(0)` you can use `x.getPropertyFirstValues("Trajector")`. 
To access the properties added from other tags use the tag name followed by an underscore and then the property name, e.g. `TRAJECTOR_id`